.changeset/cuddly-queens-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@kadena/client-utils': patch
+---
+
+export describeModule
```

Comment:
Is this a changelog that can be understood by the consumer?

Reply:
could you please check/approve the improved description of the change?